stats/view/view_to_metric.go

```diff
@@ -91,18 +91,25 @@ func viewToMetricDescriptor(v *View) *metricdata.Descriptor {
 	}
 }
 
-func toLabelValues(row *Row) []metricdata.LabelValue {
+func toLabelValues(row *Row, expectedKeys []string) []metricdata.LabelValue {
 	labelValues := []metricdata.LabelValue{}
-	for _, tag := range row.Tags {
-		labelValues = append(labelValues, metricdata.NewLabelValue(tag.Value))
+outer:
+	for _, key := range expectedKeys {
+		for _, tag := range row.Tags {
```

Comment:
Nit: you can convert `row.Tags` to a `map[string]string` to simplify this.

Reply:
done.